File: 프로그래머스/unrated/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    
    for q in quiz:
        str = q.split(" ")
        if str[1] =='-':
            if (int)(str[4]) == (int)(str[0]) - (int)(str[2]):
                answer.append("O")
            else:
                answer.append("X")
        if str[1] =='+':
            if (int)(str[4]) == (int)(str[0]) + (int)(str[2]):
                answer.append("O")
            else:
                answer.append("X")        
    
    return answer

File: 프로그래머스/0/120907. OX퀴즈/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
        StringBuffer answer = new StringBuffer();
        
        for(int i=0;i<quiz.length;i++){
            String[] quizOX = quiz[i].split(" ");
            if(quizOX[1].equals("+")){
                if(Integer.parseInt(quizOX[0]) + Integer.parseInt(quizOX[2]) == Integer.parseInt(quizOX[4])){
                    answer.append("O");
                }
                else{
                    answer.append("X");
                }
            }
            else{
                if(Integer.parseInt(quizOX[0]) - Integer.parseInt(quizOX[2]) == Integer.parseInt(quizOX[4])){
                    answer.append("O");
                }
                else{
                    answer.append("X");
                }
            }
        }
        
        
        return answer.toString().split("");
    }
}